Sources/XCHammer/XcodeTarget.swift

```diff
@@ -1077,9 +1083,11 @@ public class XcodeTarget: Hashable, Equatable {
         // fuse rules_ios: because of mixed-module output types. e.g.
         // $NAME_swift and $NAME_objc cannot both declare a .swiftmodule output
         // Could test for that, or even test for a tag
-        return type.contains("-test") ||
-            type.contains("framework") ||
-            type.contains("application")
+        return type.hasSuffix("unit-test") ||
+            type.hasSuffix("ui-testing") ||
+            type.hasSuffix("framework") ||
+            type.hasSuffix("framework.static") ||
+            type.hasSuffix("application")
```

Comment:
This is one of the key changes here right? The `contains` call was expensive (now with more stuff coming in from `rules_ios`)

Reply:
It was, but then I refactored the `flattenedInner` (which made all these calls) so it only gets called once instead of N or more times. I left this here since it's still technically faster, even if it's only a marginal improvement now.